src/ultils/status.js

```diff
@@ -1,13 +1,13 @@
-import listItems from './data.js';
 import displayListItems from './template.js';
+import Todo from './Todo.js';
 
 const inputList = displayListItems();
 
 function userActions() {
   inputList.forEach((item) => {
     item.addEventListener('change', () => {
       item.listItem.completed = !item.listItem.completed;
-      localStorage.setItem('listitems', JSON.stringify(listItems.data));
+      localStorage.setItem('todoList', JSON.stringify(Todo.data));
     });
   });
 }
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ In your `README` file the image is broken kindly fix it.